docs/server/source/quickstart.md

```diff
@@ -8,7 +8,7 @@ A. Install MongoDB as the database backend. (There are other options but you can
 
 B. Run MongoDB. Open a Terminal and run the command:
 ```text
-$ mongod --replSet=bigchain-rs
+$ sudo mongod --replSet=bigchain-rs
```

Comment:
Hmmm ... I don't know. With docker, we run it as `root` by default (without the explicit `sudo`).

This is a MongoDB issue in any case, and ideally we could simply point the user to the appropriate documentation.

The MongoDB docs I quickly looked up does not mention the usage of `sudo`: https://docs.mongodb.com/manual/tutorial/manage-mongodb-processes/

If we give instructions I guess we can mention two alternatives:

* changing the file permissions of `/data/db` (`sudo chown u+w /data/db`).
* specifying a different data directory (e.g.: `mongod --dbpath /srv/mongodb/`).